{{cookiecutter.plugin_name}}/README.md

```diff
@@ -4,87 +4,70 @@
 
 This `nomad` plugin was generated with `Cookiecutter` along with `@nomad`'s [`cookiecutter-nomad-plugin`](https://github.com/FAIRmat-NFDI/cookiecutter-nomad-plugin) template.
 
-
 ## Development
 
-If you want to develop locally this plugin, clone the project and in the plugin folder, create a virtual environment (you can use Python 3.9, 3.10, or 3.11):
+If you want to locally develop this plugin, clone the project.
+
 ```sh
 git clone https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.plugin_name}}.git
 cd {{cookiecutter.plugin_name}}
-python3.11 -m venv .pyenv
-. .pyenv/bin/activate
 ```
 
-Make sure to have `pip` upgraded:
-```sh
-pip install --upgrade pip
-```
+We highly recommend using `uv` to manage the project. Consider installing `uv` by following the official installation [guide](https://docs.astral.sh/uv/getting-started/installation/#installing-uv).
 
-We recommend installing `uv` for fast pip installation of the packages:
-```sh
-pip install uv
-```
+Read the short [guide](https://docs.astral.sh/uv/guides/projects/) on how to manage projects.
 
-Install the `nomad-lab` package:
-```sh
-uv pip install '.[dev]' --index-url https://gitlab.mpcdf.mpg.de/api/v4/projects/2187/packages/pypi/simple
-```
+Install all the dependencies:
 
-**Note!**
-Until we have an official pypi NOMAD release with the plugins functionality make
-sure to include NOMAD's internal package registry (via `--index-url` in the above command).
-
-The plugin is still under development. If you would like to contribute, install the package in editable mode (with the added `-e` flag):
 ```sh
-uv pip install -e '.[dev]' --index-url https://gitlab.mpcdf.mpg.de/api/v4/projects/2187/packages/pypi/simple
+uv sync --extra-all
```

Comment:
Ok, this is new and I wrote a comment before regarding the tool.uv part with the index-url. So maybe you can ask Bernadette if this works for her? And maybe others in the team?